llama-index-integrations/llms/llama-index-llms-vertex/llama_index/llms/vertex/base.py

```diff
@@ -495,7 +495,7 @@ def get_tool_calls_from_response(
 
         tool_selections = []
         for tool_call in tool_calls:
-            response_dict = MessageToDict(tool_call._pb)
+            response_dict = MessageToDict(tool_call._raw_message._pb)
```

Comment:
no way to do this without accessing private attributes hey? 😅 Oh google

Reply:
Well, in another branch, I did it by just creating an entirely new object by hand. I kept it this way to minimize the change surface area. I can move over to the other change style if it is an issue, but yeah- their MessageToDict function doesn't work without private access. Kind of silly; but then again, I'm not entirely sure that google makes apis for use outside of google.